test/palindrom_test.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai'
+import isPalindrome from '../src/palindrome'
+
+describe('isPalindrome()', function(){
+
+  it('should be a function', function(){
+    expect(isPalindrome).to.be.a('function')
+  })
+
+  it('Determine if a string is a palindrome. Return true or false.', function(){
+    expect(isPalindrome('radar')).to.be.true
+    expect(isPalindrome('A man, a plan, a canal: Panama')).to.be.true
+    expect(isPalindrome('bananas')).to.be.false
+  })
```

Comment:
According to https://github.com/GuildCrafts/web-development-js/blob/master/contracts/123-Core_Algorithms.md, there should be a test with an invalid input.